### PR TITLE
App views feature

### PR DESCRIPTION
**Pull request by mwenyoa**
- Implemented the design from the sneak peek wireframes.
  - Styling is nice to have but focus on making sure everything functions first. (Note: Ignore login logic, you would be using Devise in coming milestone)

- Use methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).
- **Hint:** In order to see some posts or comments, you might need to add some data to your database as described [here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md#use-models-to-insert-data).

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,5 +1,10 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = User.order(:name)
+  end
 
-  def show; end
+  def show
+    @user = User.find(params[:id])
+    @recent_posts = Post.where(author_id: params[:id]).order(created_at: :desc).limit(3)
```
Comment (Kingstalux) [inline app/controllers/users_controller.rb]: - [ ] Great job so far :+1: Kindly use the `recent_posts` method you created in the user model to display the 3 most recents posts :100: 
- Requirements:
Screenshot from 2022-08-17 16-48-41

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -1,5 +1,10 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find(params[:user_id])
+    @user_posts = Post.where(author_id: params[:user_id]).order(created_at: :desc).limit(5)
```
Comment (Kingstalux) [inline app/controllers/posts_controller.rb]: - [ ] Same issue here. Kindly use the `recent_comments` method found in the post model to display the comments :+1:

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @mwenyoa, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :clap: 
:ballot_box_with_check: App has the correct functionality :fire: 
:ballot_box_with_check: All views are present :+1: 
:ballot_box_with_check: Good commit history :clap: 
:ballot_box_with_check: Linters are passing :ok_hand: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**